crates/egui/src/data/output.rs

```diff
@@ -18,7 +19,7 @@ pub struct FullOutput {
     /// duration elapses. when in reactive mode, egui spends forever waiting for input and only then,
     /// will it repaint itself. this can be used to make sure that backend will only wait for a
     /// specified amount of time, and repaint egui without any new input.
-    pub repaint_after: std::time::Duration,
+    pub repaint_after: Vec<(ViewportId, std::time::Duration)>,
```

Comment:
I think it makes sense to have at most one duration per viewport

```suggestion
    pub repaint_after: BTreeMap<ViewportId, std::time::Duration>,
```

Reply:
I used a `HashMap` because i'am not interested in the order and because in `Repaint` was stored as a `HashMap` now i only call clone!